release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -57,6 +57,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-security"]
 === Security and compliance
 
+[id="ocp-4-8-security-log-oauth-tokens"]
+==== Audit logging for OAuth access token logout requests
+
+The `Default` audit log policy now logs request bodies for OAuth access token login and logout requests. Previously, logout request bodies were not logged.
```

Comment:
would be more specific:

... now logs request bodies for OAuth access token creation (login) and deletion (logout) requests

Reply:
Thanks, updated. I'll also make this clarification in the other PR (#32053) for pre-4.8 that gets specific about only logging logins.